### PR TITLE
Remove copied sources from analysis

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: src/Compatibility.StorageTable.AcceptanceTests/Previous/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+We looked at several solutions like
+
+- Referencing the previous nuget package leveraging aliasing
+- Using a dedicated repository to verify backward compatibility
+- Spinning up custom app domains that load different assemblies
+
+One of the main goals was to make sure we are backward compatible and detect it early on master so a dedicated repository was ruled out. Custom aliasing didn't work out in the same acceptance test project and spinning up custom app domains was deemed too complex so we settled with a tradeoff of copying the previous saga persister code into master. In order for the code to not trigger analysis the files have been treated as external sources and use `.g.cs` as a file extension.
+
+The sources used only need to be updated if a patch version on the 2.4 branch introduced changes to the `DictionaryTableEntity` or the secondary index algorithm.
```
Comment (SeanFeldman) [inline src/Compatibility.StorageTable.AcceptanceTests/Previous/README.md:9]: Makes me wonder if the Previous folder should include’release-2.4’ in the name to be more explicit and have a text/md note there.

**Review by SeanFeldman — APPROVED** [review]
With a note

File: src/Compatibility.StorageTable.AcceptanceTests/Previous/README.md (hunk 1 above)
Comment (danielmarbach) [reply src/Compatibility.StorageTable.AcceptanceTests/Previous/README.md:9]: What if we call the folder PreviousMajor?

Comment (SeanFeldman) [reply src/Compatibility.StorageTable.AcceptanceTests/Previous/README.md:9]: 👍

**Review by lailabougria — APPROVED** [review]

**Pull request merged**